sdk/storage/azure-storage-queue/tests/queuetestcase.py

```diff
@@ -69,38 +44,15 @@ def __init__(self):
     def get_token(self, *args):
         return self.token
 
+class QueueTestCase(AzureMgmtTestCase):
+    def connection_string(self, account, key):
+        return "DefaultEndpointsProtocol=https;AccountName=" + account.name + ";AccountKey=" + str(key) + ";EndpointSuffix=core.windows.net"
 
-class QueueTestCase(unittest.TestCase):
-
-    def setUp(self):
-        self.working_folder = os.path.dirname(__file__)
-
-        self.settings = settings
-        self.fake_settings = fake_settings
-
-        if settings is None:
-            self.test_mode = os.getenv('TEST_MODE') or TestMode.playback
-        else:
-            self.test_mode = self.settings.TEST_MODE.lower() or TestMode.playback
-
-        if self.test_mode == TestMode.playback or (self.settings is None and self.test_mode.lower() == TestMode.run_live_no_record):
-            self.settings = self.fake_settings
-
-        # example of qualified test name:
-        # test_mgmt_network.test_public_ip_addresses
-        _, filename = os.path.split(inspect.getsourcefile(type(self)))
-        name, _ = os.path.splitext(filename)
-        self.qualified_test_name = '{0}.{1}'.format(
-            name,
-            self._testMethodName,
-        )
-
-        self.logger = logging.getLogger('azure.storage')
-        # enable logging if desired
-        self.configure_logging()
-
-    def configure_logging(self):
-        self.enable_logging() if self.settings.ENABLE_LOGGING else self.disable_logging()
+    def _get_oauth_queue_url(self, name):
+        return "https://{}.queue.core.windows.net".format(name)
+    
+    def _account_url (self, name):
```

Comment:
Looks like this and the function above can be refactored into a simple function.

Reply:
🥇 for my stupidity